8.observability/deploy/python-multiplier.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: multiplyapp
+  labels:
+    app: multiply
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: multiply
+  template:
+    metadata:
+      labels:
+        app: multiply
+      annotations:
+        dapr.io/enabled: "true"
+        dapr.io/id: "multiplyapp"
+        dapr.io/port: "5000"
+        dapr.io/config: "tracing"
+    spec:
+      containers:
+      - name: multiply
+        image: orizohar/distributed-calculator-slow-python:edge
```

Comment:
please switch to `dapriosamples` hub

Reply:
Good catch! I was worried I'd do that and here we are :)

Reply:
Resolved via https://github.com/dapr/samples/pull/183/commits/5e289768d44af7d10e4ee2164aa101633269ade4